examples/structured_data/classification_with_tfdf.py

```diff
@@ -0,0 +1,726 @@
+"""
+Title: Classification with TensorFlow Decision Forests
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2022/01/31
+Last modified: 2022/01/31
+Description: Using TensorFlow Decision Forests for structured data classification.
```

Comment:
Nit: This colab is about two advanced ideas: Target mean and composition of Linear embedding and Decision Forests. Should the title reflect this?

Also, what about linking the TF-DF introductory colab (https://www.tensorflow.org/decision_forests/tutorials/beginner_colab), and mention that it is a recommended pre-requirement for this colab?